charters/DataOperations/charter.md

```diff
@@ -0,0 +1,92 @@
+
+## In-scope
+
+- Data operations
+  - Engage with all DCP components to identify deliverables, including data, pipelines, metadata, standards, and user features, which maximize the value and consistency of HCA data
+  - Coordinate DCP-wide end-to-end testing pre-release and continued QC checks post-release to ensure data and data resource integrity and useability through component updates
+  - Identify data release- and data distribution-associated engineering needs of all DCP components and participate in crafting technical specifications
+  - Establish and integrate into the DCP roadmap a reproducible plan that includes the specifications for and the timing of iterative data distribution to the community
+  - Monitor the status of data contributions at each step in the DCP data workflow from initial identification of suitability through data release and potential inclusion in additional data distribution
+  - Engage with Governance Group and Analysis Work Group to identify data quality standards which dictate data eligible to be released and data eligible to be included in data distributions
+  - Engage with the [Compliance Working Group](https://github.com/HumanCellAtlas/dcp-community/blob/master/charters/Compliance-WG/charter.md) to ensure that the data release and data distribution processes are facilitating appropriate data access to all of the types of consumers
+  - Ensure released data are useable to tertiary portals and users can easily cross-reference across portals
+  - Determine best practices for when and how changes are made to DCP outputs after release
+
+- Community engagement
+  - Define data use policy and citation instructions for HCA data users
+  - Define mechanism(s) for which the community is notified when DCP products are updated after release
+  - Partner with Content and Portal Development to ensure clear, findable, and current documentation describing data release and data distribution
+  - Ensure effective communication of HCA data updates and user features to the scientific community
+  - Partner with UX to assess curation added value of data distribution through communities’ feedback, and better understand their expectations and needs
+  - Onboard staff to DCP ZenDesk ticketing system
+  - Take ownership of establishing process and interfacing with DCP components for assigning, responding, and integrating ZenDesk tickets
+  - Compile feedback gained from consumers through ZenDesk queries and comments, and report back to the DCP in order to facilitate system improvement 
+
+
+
+### Scientific "guardrails"
+
+In order to meet the needs of the communities of scientific consumers, the DataOps team requires input from the Governance Group as to where curated data distribution will add value for the researcher.
+
+## Out-of-scope
+
+The Data Operations is not responsible for determining what data or data types are made available through the HCA.
```

Comment:
That statement seems in conflict with the statement about data suitability

Reply:
I believe this was clarified by rewording the statement about data suitability. Resolving this conversation.